scripts/extract-replaygain

```diff
@@ -2,22 +2,42 @@
 
 set -e
 
-FFMPEG=`which ffmpeg || echo ""`
+FFMPEG=`which ffmpeg 2>/dev/null || echo ""`
+FFPROBE=`which ffprobe 2>/dev/null || echo ""`
 
 if test -z "$FFMPEG"; then
-  echo "ffmpeg must be installed"
-  exit 1
+        echo "ffmpeg must be installed"
+        exit 1
```

Comment:
Why change the indentation here?
```suggestion
  echo "ffmpeg must be installed"
  exit 1
```

Reply:
Sorry for the typos thank for correcting this.